superset/assets/src/visualizations/PairedTTest/PairedTTest.jsx

```diff
@@ -0,0 +1,83 @@
+
+import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
+import React from 'react';
+import TTestTable from './TTestTable';
+import './PairedTTest.css';
+
+const propTypes = {
+  className: PropTypes.string,
+  metrics: PropTypes.arrayOf(PropTypes.string).isRequired,
+  groups: PropTypes.array.isRequired,
+  data: PropTypes.object.isRequired,
```

Comment:
could we be more specific about the shape here? I can't quite figure out what groups is from the code, seems like an array based on [this line](https://github.com/apache/incubator-superset/pull/5762/files#diff-34f99612972b2818167471a47a5559acR140)

`{ my_metric: [ { group: [array of something?] values: [number, number] } ] }`

```javascript
PropTypes.objectOf(
  PropTypes.arrayOf(
    PropTypes.shape({ 
      group: PropTypes.arrayOf(??), 
      values: PropTypes.arrayOf(PropTypes.number), 
    }),
  ),
)
```